src/topp/QualityControl.cpp

```diff
@@ -172,6 +174,13 @@ class TOPPQualityControl : public TOPPBase
       FASTAFile().load(in_contaminants, contaminants);
       status |= QCBase::Requires::CONTAMINANTS;
     }
+
+    //the additional file the user passed, with annotate genenames & protnames
+    String test_file = getStringOption_("in_additional_info");
+    vector<FASTAFile::FASTAEntry> prot_description;
+    OPENMS_LOG_INFO << "Loading FASTA ... " << test_file << std::endl;
+    FASTAFile().load(test_file, prot_description);
```

Comment:
test_file koennte leer sein...
und dann kracht es in Zeile 182?
Fix siehe unten?